### ISSUE TITLE
Integrating checkout.js with AngularJS

### ISSUE THREAD

**Issue by MTShare** [paypal/paypal-checkout#126]
I followed the implementation tutorial to use the checkout.js (v.4) in conjunction of AngularJS but i keep getting this:
`Error: $injector:modulerr
Module Error
Failed to instantiate module paypal-button due to:
Error: [$injector:nomod] http://errors.angularjs.org/1.5.9/$injector/nomod?p0=payp...`

Any suggestions?

**Comment by MTShare** [comment]
Now keep saying:
`Error: [paypal-button] No context options available for render`
Even if i added the props needed to my $scope

**Comment by bluepnume** [comment]
Yeah, I've managed to repro this. Seems to be a regression in the angular driver. Should be a straightforward fix, I'll let you know shortly.

**Comment by MTShare** [comment]
Thanks a lot!! ;)

**Comment by bluepnume** [comment]
@marcocristalli I've pushed a fix, please make sure your browser cache is cleared and try again.

There's a working demo at https://github.com/paypal/paypal-checkout/blob/master/demo/angular.htm for reference.

**Comment by marcocristalli** [comment]
Awesome! Now it works, thanks a lot! ;)

**Comment by usamamalik75** [comment]
@bluepnume its working fine but one issue here when payment successful all object are undefined 
mean here Router undefined
this.router.navigate(['/confirmation', this.Order.id]);

```

>  onAuthorize: function(data, actions) {
>                     // Optional: display a confirmation page here
>                     return actions.payment.execute().then(function() {
>                         // this.router.navigate(['/confirmation', this.Order.id]);
>                     });
>                 }

```
Can you please what a problem here why router object show undefined?
